packages/fxa-payments-server/src/routes/Subscriptions/index.scss

```diff
@@ -184,18 +184,7 @@
 // Wipe out button/link styles so that we can reuse the behavior from the
 // manage/change button inside of the alert bar
 
-// Alerts are going to be refactored
-#top-bar .alert {
-  margin: 0;
-}
-
-.alert {
-  margin: 0;
-
-  .action {
-    display: inline;
-  }
-
+.alert-error {
```

Comment:
I believe `alert` can be updated to `alert-error` here, as these styles look like they are used in ActionButton only. I initially thought this would be an easy update to include in this PR; however, as I tried to unravel styles in `ActionButton`, I noticed there are classes being used elsewhere (e.g., `settings-button`), and I'd prefer to update this in another PR, especially with the comment that was made above on L184 (`Wipe out button/link styles so that we can reuse the behavior from the manage/change button inside of the alert bar`).